js/islandora_internet_archive_bookreader.js

```diff
@@ -35,6 +35,10 @@ Drupal.settings.islandoraInternetArchiveBookReader_jQuery = jQuery.noConflict(tr
         if ($.browser.mobile && settings.islandoraInternetArchiveBookReader.mobilize) {
           bookReader.goFullScreen();
         }
+        // Add class 'aria-label' for accessibility support for screen readers.
```

Comment:
This should go, just in case (picky picky), before the fullscreen activation.

Reply:
Moving this above mobile browser detection.